components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -92,9 +92,10 @@
  functionality and embedded frameworks that also render UI.
 
  @warning This API will eventually be deprecated. Please use
- @c useCurrentContentSizeCategoryWhenApplied instead.
+ @c useCurrentContentSizeCategoryWhenApplied instead. It is recommended to use
+ @c useCurrentContentSizeCategoryWhenApplied as the backing value during deprecation.
```

Comment:
I don't think it should be a recommendation, but a requirement. I also think it should be clear that the values should be the same even before it's deprecated. Suggestion: 

```
@note  The value of @c useCurrentContentSizeCategoryWhenApplied (if implemented) and @c mdc_adjustsFontForContentSizeCategory must always be the same.
```

Reply:
Addressed in b63d4be9642aeac93853b15e32cfb2e6c7477fcb